src/Datadog.Trace.Tools.Runner/Datadog.Trace.Tools.Runner.proj

```diff
@@ -0,0 +1,167 @@
+﻿<Project DefaultTargets="BuildTool" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <TracerHomeDirectory Include="$(MSBuildThisFileDirectory)home" />
+    <CSharpProjects Include="..\**\*.csproj"/>
+    <NativeWinProject Include="..\**\*.vcxproj" />
+    <NativeWinX64OutputFolder Include=".\home\win-x64" />
+    <NativeWinX86OutputFolder Include=".\home\win-x86" />
+    <NativeLinuxX64Output Include="$(MSBuildThisFileDirectory)home\linux-x64\Datadog.Trace.ClrProfiler.Native.so" />
+    <ObjPath Include="obj\**\*" />
+  </ItemGroup>
+
+  <Target Name="BuildTool" DependsOnTargets="BuildCSharpProjects;BuildNativeWindows;BuildNativeLinux">
+    <Message Text="Building the dotnet tool" />
+    <Delete Files="@(ObjPath)" />
+    <RemoveDir Directories="obj" />
+    <MSBuild Targets="Restore" Projects="Datadog.Trace.Tools.Runner.Tool.csproj" />
+    <MSBuild Targets="Build" Projects="Datadog.Trace.Tools.Runner.Tool.csproj" Properties="Configuration=Release" />
+  </Target>
+
+  <Target Name="BuildStandalone" DependsOnTargets="BuildCSharpProjects;BuildNativeWindows;BuildNativeLinux">
+    <Message Text="Building the standalone app" />
+    <Delete Files="@(ObjPath)" />
+    <RemoveDir Directories="obj" />
+    <MSBuild Targets="Restore" Projects="Datadog.Trace.Tools.Runner.Standalone.csproj" />
+    <MSBuild Targets="Publish" Projects="Datadog.Trace.Tools.Runner.Standalone.csproj" Properties="Configuration=Release;TargetFramework=netcoreapp3.1;RuntimeIdentifier=win-x64" />
+    <MSBuild Targets="Publish" Projects="Datadog.Trace.Tools.Runner.Standalone.csproj" Properties="Configuration=Release;TargetFramework=netcoreapp3.1;RuntimeIdentifier=win-x86" />
+    <MSBuild Targets="Publish" Projects="Datadog.Trace.Tools.Runner.Standalone.csproj" Properties="Configuration=Release;TargetFramework=netcoreapp3.1;RuntimeIdentifier=linux-x64" />
+  </Target>
+
+  <Target Name="BuildCSharpProjects">
+    <Error Text="To build this tool a Windows machine is required." Condition="'$(OS)' != 'Windows_NT'" />
```

Comment:
How comes Windows is required here? I figured only the Native Windows part would require this

Reply:
Honestly I have my doubts compiling for multiple versions of the old framework from a non windows machine. If you tell me that is ok, I’ll remove this condition.

Reply:
Oh yeah, I forgot the managed targets will compile for .NET Framework. I'm not 100% sure it is safe so let's leave this in.

Reply:
You can target .NET Framework just fine from Linux, but for now you need to include [the reference libraries](https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies/) like we do [here](https://github.com/DataDog/dd-trace-dotnet/blob/9d1f0e380cc6d2b9964b02aec7fce5f356bbe4e1/src/Directory.Build.props#L32).

.NET 5 SDK was supposed to include them automatically like current SDKs do for .NET Core targets. I don't know if that feature made it in.

Reply:
Oh that's great, so I'll remove the condition and add the nuget reference then.

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/962/commits/b230ad3828272e50a8d366216b43d814b15b7983